tests/test_modules.py

```diff
@@ -65,43 +73,43 @@ def test_modulesrepo_class(self):
     # Test of the individual modules commands. #
     ############################################
 
-    from modules.list import (
+    from .modules.list import (
```

Comment:
A relative import is better: this is required to call `pytest --fixtures` without errors